### PR TITLE
Created the home-top-bar directive

### PR DESCRIPTION
**Pull request by ghost**
**TICKET:**
https://trello.com/c/ljxDOlDF/3-start-skeleton

**PURPOSE:**
Foundation for home's top bar camera thing. 

**SETUP:**
npm start

**HOW TO TEST:**
Go to /home and see if the camera icon comes up. 

**SCREENSHOTS:**
image

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -35,7 +35,7 @@
     "angular/on-watch": 1,
     "angular/no-http-callback": 1,
     "angular/controller-name": 2,
-    "angular/directive-name": [2,"ci"],
+    "angular/directive-name": 0,
```
Comment (ghost) [inline .eslintrc:38]: Name your directives what you want.

File: app/app.module.js [hunk 2]
```diff
@@ -1,10 +1,11 @@
+require('./main.scss');
+require('../node_modules/bootstrap-sass/assets/stylesheets/_bootstrap.scss');
```
Comment (ghost) [inline app/app.module.js:2]: Hooking up bootstrap

File: app/home/home-config.js [hunk 3]
```diff
@@ -1,7 +1,4 @@
 function homeState($stateProvider, $urlRouterProvider) {
-    $urlRouterProvider.when('', '/home');
```
Comment (ghost) [inline app/home/home-config.js:2]: this is duplicate code.

File: app/home/home-controller.js [hunk 4]
```diff
@@ -1,7 +1,4 @@
 function homeController() {
-    return {
```
Comment (ghost) [inline app/home/home-controller.js:2]: controllers never return.

**Review by Ashkash28 — APPROVED** [review]

**Review by parakhkunal — APPROVED** [review]

**Pull request merged**